File: MiniDOMAsteriskLib/CallManagers/Events/Leave.cs

namespace minidom.CallManagers.Events
{
    public class Leave : AsteriskEvent
    {
        public Leave() : base("Leave")
        {
        }

        public Leave(string[] rows) : base(rows)
        {
        }

        public string Channel
        {
            get
            {
                return GetAttribute("Channel");
            }
        }

        public string Queue
        {
            get
            {
                return GetAttribute("Queue");
            }
        }

        public int Count
        {
            get
            {
                return GetAttribute("Count", 0);
            }
        }
    }
}

File: MiniDOMAsteriskLib/CallManagers/Events/Leave.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net

Namespace CallManagers.Events

    Public Class Leave
        Inherits AsteriskEvent

        Public Sub New()
            MyBase.New("Leave")
        End Sub

        Public Sub New(ByVal rows() As String)
            MyBase.New(rows)
        End Sub

        Public ReadOnly Property Channel As String
            Get
                Return Me.GetAttribute("Channel")
            End Get
        End Property

        Public ReadOnly Property Queue As String
            Get
                Return Me.GetAttribute("Queue")
            End Get
        End Property
         
        Public ReadOnly Property Count As Integer
            Get
                Return Me.GetAttribute("Count", 0)
            End Get
        End Property
         
    End Class

End Namespace